vuu-ui/showcase/src/examples/Layout/SavePanel.examples.css

```diff
@@ -0,0 +1,13 @@
+.dialogHeader {
+  display: flex;
+  padding-bottom: 0px;
+  padding-top: 16px;
+  padding-left: 16px;
+  flex-direction: column;
+  align-items: flex-start;
+  align-self: stretch;
+  color: var(--light-text-primary, #15171B);
+  font-feature-settings: 'ss02' on, 'ss01' on, 'salt' on, 'liga' off;
+  font-size: 16px;
+  font-weight: 600;
+}
```

Comment:
Can you think of a scenario where the dialog header will need to be styled differently?

Reply:
There are other header styles in Figma (see screenshot), but I'd imagine we'd want to select a type from a list of presets rather than allowing custom styling. I don't think we need to introduce such options at this stage though, so I have removed the `headerProps`.

![image](https://github.com/ScottLogic/finos-vuu/assets/79100986/f9e77f20-7cd7-4989-9aa1-c11dfb1d7d99)

Reply:
Ahhhh right okay, yes that makes sense